x-pack/plugins/fleet/server/routes/agent_policy/handlers.ts

```diff
@@ -677,3 +685,72 @@ export const downloadK8sManifest: FleetRequestHandler<
     return defaultFleetErrorHandler({ error, response });
   }
 };
+
+export const GetAgentPolicyOutputsHandler: FleetRequestHandler<
```

Comment:
Handler for the new endpoint - I still need to work on it, it's in a very basic state at the moment